LuizWeitz.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "@LuizWeitz"
+project: "E-reader using Raspberry Pi Pico"
+---
+
+# E-reader With Raspberry Pi Pico
+
+## Summary
+
+E-reader using Raspberry Pi Pico with E-Ink technology screen with rechargeable battery that fits in the palm of the hand, through which I intend to expand my adventures in the world of reading.
+
+## Plan
+
+The first step is to plan how the components will be organized. <br>
+The second step is to connect all components through wire and solder.  <br>
+The third step is to plan a simple case in 3d and print out. <br>
+The fourth step is to create a simple software that reads the books and sends them to the screen with the possibility of changing pages.  <br>
+The fifth step is to test everything.  <br>
+
+### one of the prototypes:
+
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+|  Raspberry Pi Pico Board | https://www.aliexpress.com/item/1005002890682924.html?spm=a2g0o.cart.0.0.267538daPxA5wQ&mp=1 | $9.58 |
+| Waveshare 7.5inch E-Paper E-Ink Display Module For Raspberry Pi Pico | https://www.aliexpress.com/item/1005002841576759.html?spm=a2g0o.cart.0.0.267538daPxA5wQ&mp=1 | $63.89 |
+| MicroSD Card Adapter| https://www.aliexpress.com/item/1005001309671718.html?spm=a2g0o.cart.0.0.267538daPxA5wQ&mp=1 | $6.37 |
+| Lexar Card SD 32GB | https://www.aliexpress.com/item/1005003780732823.html?spm=a2g0o.cart.0.0.267538daPxA5wQ&mp=1 | $6.04 |
```

Comment:
this is a personal comment and it won't affect your acceptance, but I'd suggest against getting SD cards from AliExpress, as they're ofter faulty. Maybe try Amazon or a local retailer?

Reply:
Hi, So the store where I'm getting the sd card is the official Lexar store, I have their sd card that until today has not presented any problems